lldb/source/Plugins/Process/Windows/Common/ProcessWindows.h

```diff
@@ -52,7 +52,7 @@ class ProcessWindows : public Process, public ProcessDebugger {
   Status DoAttachToProcessWithID(
       lldb::pid_t pid,
       const lldb_private::ProcessAttachInfo &attach_info) override;
-  Status DoResume() override;
+  Status DoResume(lldb::RunDirection direction) override;
```

Comment:
We should use `DoResumeReverse()` in Process.h and this change won't need to happen.

Reply:
The decision was to use a direction flag.